### PR TITLE
Try to solve the stacking context issue using either `<dialog>` or `popover`

### PR DESCRIPTION
**Pull request by florianstancioiu**
- Try to solve the stacking context issue using the dialog element.
- Try to solve the stacking context issue using the popover attribute.

### CODE REVIEW

File: src/cosmoz-dropdown.ts [hunk 1]
```diff
@@ -43,6 +45,21 @@ interface DropdownProps
 		Pick<ContentProps, 'placement' | 'render'> {}
 
 const Dropdown = (host: HTMLElement & DropdownProps) => {
+	const supportsPopover = () => {
+		// eslint-disable-next-line no-prototype-builtins
+		return HTMLElement.prototype.hasOwnProperty('popover');
+	};
```
Comment (megheaiulian) [inline src/cosmoz-dropdown.ts:51]: Move this outside Content.

File: src/cosmoz-dropdown.ts [hunk 2]
```diff
@@ -43,6 +45,21 @@ interface DropdownProps
...
+
+	const popoverChanged = (popover?: Element) => {
```
Comment (megheaiulian) [inline src/cosmoz-dropdown.ts:53]: Move this outside Content and rename it to showPopover.
 Do not even request the animation frame if popover is not supported.

File: src/cosmoz-dropdown.ts [hunk 3]
```diff
@@ -104,16 +128,17 @@ const Dropdown = (host: HTMLElement & DropdownProps) => {
...
-					id="content"
-					part="content"
-					exportparts="wrap, content"
-					.anchor=${anchor}
-					.placement=${placement}
-					.render=${render}
-				>
-					<slot></slot>
-				</cosmoz-dropdown-content>`
+				html`<div ${ref(popoverChanged)} popover id="content">
```
Comment (megheaiulian) [inline src/cosmoz-dropdown.ts:131]: Can't we set the `popover`  attribute on the `cosmoz-dropdown-content` element and avoid creating another div  ?

**Review by megheaiulian — CHANGES_REQUESTED** [review]

File: src/cosmoz-dropdown.ts (hunk 3 above)
Comment (florianstancioiu) [reply src/cosmoz-dropdown.ts:131]: I tried just now, but when I click the anchor nothing shows up.

Comment (florianstancioiu) [reply src/cosmoz-dropdown.ts]: I think I managed to get it right in commit 7dcc348

File: src/cosmoz-dropdown.ts [hunk 4]
```diff
@@ -90,6 +107,15 @@ const Dropdown = (host: HTMLElement & DropdownProps) => {
 					left: auto;
 				}
 			}
+			#content:popover-open {
```
Comment (megheaiulian) [inline src/cosmoz-dropdown.ts:110]: Try to use :host(:popover) in the Content component itself.

**Review by megheaiulian — APPROVED** [review]

**Review by cristinecula — APPROVED** [review]

**Pull request merged**